### DISCUSSION TITLE
educational discount request rejected

### DISCUSSION THREAD

**Discussion by Kevin65536** [community/community#51208] [GitHub Education]
I uploaded these documents and all rejected.

- School ID card: response is "The image you selected does not appear to contain your school name. ". But there is a logo of my school in the background of the card. I've tried to mark it with English translation, but nothing changed. 
- Enrollment Certificate: I uploaded an Enrollment Certificate from my school's Students Registration Center. It contains my full name which matches with my name in GitHub profile, the date I was matriculated, my expected graduate date, and it is on a school letter head. In addition, it mentioned how to verify its authticity. This document is in English. The reject information I received is "Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved."  
But my situation does not correspond to any of the reasons mentioned on that page. 
- Online verification report of the Ministry of Education: This is a certification given by the Ministry of Education. It includes my full name, my school my enrollment date and my student status. It also contains information about when this certification was given. It can be verified by using the online verification code in the document. I tried to upload this document's Chinese and English version. At first, I was rejected because "You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio." But that time I have changed my user profile, with my full name which matches with the name in the report and my short bio about my school and my enrollment. After two tries of uploading this document, the respond changed into "Please upload a different document to demonstrate your current academic status." 
- Offical transcript: I tried to upload a photo of my transcript, it is too long so I can only shoot a part of it, include the title "XXX University Transcript of Academic Record", name,  student ID NO., gender, faculty, major, admission date, and duration. This document include my photo which is the same as the photo on my school ID card and in my github profile. The reason of rejection is "The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation." But the photo I upload is already an English version and contains my school name.
- Screen shot of school portal.: I uploaded a photo of my school portal, it doesn't contain my school name, but the logo is shown on the page. My photo is also shown on this page. It was rejected because "The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation."

I have changed my GitHub profile with information that correspond to these documents. The mailbox I use is my school's mailbox with edu suffix. I've check the page which reject information directed me to, but it doesn‘t help. How can my request be approved?

**Comment by samus-aran** [comment]
Hi @Kevin65536 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
I see you said you've used a Student ID but name wasn't identified. School names are preferred rather than logos so try translating the name and ensure it's the same name as the school you've selected. 

If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.

Hope this helps. 👍

**Reply by Kevin65536** [reply]
First thank you for your answer, but as I mentioned above, the five kinds of evidence I've tried, they already include the three evidence types with highest preference. It is difficult for me to provide a picture of my school ID card with my school name in it, because it is on the other side of the card. I've also tried your suggestion of place an English translation next to the original document, but still rejected. What else should I do to pass the certification?

**Reply by samus-aran** [reply]
Hi @Kevin65536 

We understand that some Student ID have two sides. In this case I would suggest that you layer your Student ID alongside another pieces of evidence that does have the school name e.g. letterhead. Ensure that important information is seen like your details, date, school etc.

Remember the text need to be legible and match the selected school. We can detect minor differences including spaces and commas so be aware of that if you are using a logo or pre-translated copy of text.

Hope this helps! Let us know how you get along 👍

**Reply by Kevin65536** [reply]
Thanks a lot for this suggestion. But I’ve tried several times putting my Student ID on my school certificate(English version provided by students registration center) and then take the photo, but still rejected. Do you have any more suggestion that may help?

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
